packages/electron-webpack/src/dev/dev-runner.ts

```diff
@@ -136,9 +137,10 @@ main()
     console.error(error)
   })
 
-function startElectron() {
+async function startElectron() {
   const electronArgs = process.env.ELECTRON_ARGS
-  const args = electronArgs != null && electronArgs.length > 0 ? JSON.parse(electronArgs) : ["--inspect=5858"]
+  const port = await getPort();
```

Comment:
Please note that while `get-port` supports a "favorite" port number via `getPort({port: 5858})`, the typescript definitions from `@types/get-port` do not support it yet.
So, in order to make compilation possible, we currently do not specify a port and just use whatever comes back instead.